deployer/troubleshooting-integration/src/main/java/io/hyscale/troubleshooting/integration/models/AbstractedErrorMessage.java

```diff
@@ -36,7 +36,9 @@
     MULTIPLE_DEFAULT_STORAGE_CLASS("Volume creation failed. More than 1 default storage class configured on the cluster.","Provide a storage class name in the service spec or contact your cluster administrator to configure a single default storage class."),
     INVALID_RESOURCE_NAME("{} available in cluster is invalid.", "Fix the resource name and redeploy"),
     TRY_AFTER_SOMETIME("Deployment is still in progress, service is not yet ready.", "Try querying after sometime."),
-    SERVICE_COMMANDS_FAILURE("Service startup commands failed with {}" , "Possible errors in startCommands in service spec or ENTRYPOINT/CMD in Dockerfile");
+    SERVICE_COMMANDS_FAILURE("Service startup commands failed with {}" , "Possible errors can be in startCommands of hspec or ENTRYPOINT/CMD in Dockerfile"),
+    SERVICE_COMMANDS_FAILURE_WITH_UNRESOLVE_EXITCODE("Service startup commands failed with exit code {}" , "Possible errors can be in startCommands of hspec or ENTRYPOINT/CMD in Dockerfile");
```

Comment:
Can we combine these two into one?
Also instead of hspec use service spec as we are using that as a norm in other messages.